src/include/storage/write_ahead_log/log_record.h

```diff
@@ -89,16 +89,24 @@ class RedoRecord {
   MEM_REINTERPRETATION_ONLY(RedoRecord)
 
   /**
-   * @return pointer to the DataTable that this Redo is concerned with
+   * @return the tuple slot changed by this redo record
    */
-  DataTable *GetDataTable() const { return table_; }
+  TupleSlot GetTupleSlot() const { return tuple_slot_; }
```

Comment:
nit: I believe for some other fields we are returning a reference instead of having a separate setter

Reply:
I was gonna do that but TupleSlot is a type name. The signature can't be `TupleSlot &TupleSlot()`.